core/src/main/java/org/jdbi/v3/core/config/Configurable.java

```diff
@@ -100,6 +100,30 @@ default This setSqlParser(SqlParser parser) {
         return configure(SqlStatements.class, c -> c.setSqlParser(parser));
     }
 
+    /**
+     * convenience method to set the binding and mapping policy for enums to use ordinal values.
+     *
+     * @return this
+     */
+    @SuppressWarnings("unchecked")
+    default This handleEnumsByOrdinal() {
+        getConfig(Arguments.class).setBindEnumsByName(false);
+        getConfig(ColumnMappers.class).setMapEnumsByName(false);
```

Comment:
Feels like we want an `Enums` config class with a single property for this.

Reply:
Well I thought people might want to set them up to behave differently by default in read and write, like if they mainly read from one set of tables with a convention for one way, and mostly write to other tables with the other convention. That and I think `Enums` is even more of a dead-end config class than Time...

I'm 50/50 for both, doesn't really matter to me. Shall I change it?

Reply:
I also think it should be a simple property.  I don't think having read and write behavior differ makes all that much sense.